evoting/lib/chains.go

```diff
@@ -32,17 +32,15 @@ func NewSkipchain(s *skipchain.Service, roster *onet.Roster, verifier []skipchai
 // using websockets. Used for storing a block while executing a protocol.
 func StoreUsingWebsocket(id skipchain.SkipBlockID, roster *onet.Roster, transaction *Transaction) error {
 	client := skipchain.NewClient()
-	reply, err := client.GetUpdateChain(roster, id)
+	latest, err := client.GetSingleBlockByIndex(roster, id, -1)
```

Comment:
I think we should not use the `-1` and remove it. It opens up too big a hole for the server to give wrong blocks. `GetUpdateChain` is much more secure, as it checks the proof of the blocks.